<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLoginDialog</class>
 <widget class="QDialog" name="UserLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QLineEdit" name="username_line">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_line">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="company_line">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>28</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Company's ID</string>
   </property>
   <property name="buddy">
    <cstring>company_line</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
   <property name="buddy">
    <cstring>username_line</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="buddy">
    <cstring>password_line</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="logged_in_check">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Keep me logged in</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>company_line</tabstop>
  <tabstop>username_line</tabstop>
  <tabstop>password_line</tabstop>
  <tabstop>login_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>company_line</sender>
   <signal>returnPressed()</signal>
   <receiver>username_line</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>username_line</sender>
   <signal>returnPressed()</signal>
   <receiver>password_line</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>password_line</sender>
   <signal>returnPressed()</signal>
   <receiver>login_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>login_button</sender>
   <signal>pressed()</signal>
   <receiver>UserLoginDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
